tools/tertiary-analysis/garnett/garnett_macros.xml

```diff
@@ -0,0 +1,46 @@
+<macros>
+    <token name="@TOOL_VERSION@">0.2.8</token>
+    <token name="@HELP@">More information can be found at https://cole-trapnell-lab.github.io/garnett/docs/</token>
+    <token name="@PROFILE@">18.01</token>
+    <xml name="requirements">
+        <requirements>
+            <requirement type="package" version="0.0.3">garnett-cli</requirement>
```

Comment:
Why is monocle3-cli a dependency here? Also, if r-glmnet is a dependency it should be made an upstream dependency of garnett-cli. 

The reason this is important is that when we run these workflows in a containerised environment, we need a container available. Where there is a single Bioconda package, we get that anyway. But where there are multiple dependencies we have to use a separate process to create a custom container.

Reply:
@pinin4fjords, I think I wrote this before adding these packages to garnett-cli recipe. I'll check how tests work without them

Reply:
I'm guessing monocle3-cli should not be a dependency- you're not calling any of those wrappers here (unless I skimmed past). If some of the same upstream dependencies of that turn out to be necessary, you should copy them to the garnett-cli recipe.

Reply:
yep, it works fine w/o these dependencies, so I removed them